src/PrestaShopBundle/Form/Admin/Sell/CatalogPriceRule/CatalogPriceRuleType.php

```diff
@@ -126,15 +104,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     new CleanHtml(),
                 ],
             ])
-            ->add('id_currency', ChoiceType::class, [
-                'required' => false,
-                'placeholder' => false,
-                'choices' => $this->getModifiedCurrencyChoices(),
-                'choice_attr' => $this->currencyByIdChoicesAttributes,
-                'attr' => [
-                    'data-default-currency-symbol' => $this->defaultCurrencySymbol,
-                ],
-            ])
+            ->add('id_currency', CurrencyChoiceType::class)
```

Comment:
`add_all_currencies_option` is missing here, so is the data attribute But as I commented in another this one could probably be added in the default options all the time

Reply:
Updated, I missed that one